Makefile

```diff
@@ -43,10 +43,16 @@ up:
 	$(DCO_BIN) up -d
 	sleep 3
 
+# Stop the Docker containers
+.PHONY: down
+down:
+	$(DCO_BIN) down
+
 # Rebuild Docker containers to wipe all data
 .PHONY: reset
 reset:
 	$(DCO_BIN) down
+	make ent-gen
```

Comment:
This is not needed as it's used to generate the entity code files from Ent, if you changed or added entity definitions.